static/app/components/events/eventCause.tsx

```diff
@@ -72,14 +72,21 @@ export function EventCause({group, eventId, project, commitRow: CommitRow}: Prop
   };
 
   const commits = getUniqueCommitsWithAuthors();
+
+  const commitHeading =
+    commits.length > 1
+      ? `${t('Suspect Commits')} (${commits.length})`
+      : t('Suspect Commit');
+
```

Comment:
Use [`tn`](https://github.com/getsentry/sentry/blob/92bb369617d86ed62ce80861773635bce5d7b192/static/app/locale.tsx#L346-L372)

```suggestion
    tn('Suspect Commit', 'Suspect Commits (%s)', commits.length); 
```

Reply:
TIL. Can do.